src/common/components/infoBlock/InfoBlock.tsx

```diff
@@ -73,7 +73,7 @@ function InfoBlockLink({
   }
 
   return (
-    <Link href={href}>
+    <Link href={href} avoidEscaping>
```

Comment:
This one was little bit of a pain :) Needed to add avoidEspacing for the Link to work properly.

Reply:
This should be maybe applied where you use the component, e.g. in `venues/[id]/index.tsx`. Or if you prefer it as a default, then consider toggling it on by default in the `Link` component.

Reply:
I didn't want to break the default behaviour of `Link`. I doubt that it'll break anything, but there may be niche cases where the behaviour differs from that of Next.

When you toggle this prop on, it takes the `object` representation of a route and stringifies it this side instead of within the library. In the library the params get encoded, which causes issues. We don't do that in the custom logic.

That may lead to inconsistent behaviour in some cases.